### ISSUE TITLE
Performance of VTAB-DTD 

### ISSUE THREAD

**Issue by RaptorMai** [dongzelian/SSF#13]
Thank you so much for sharing the code. I tried running train_scripts/vit/vtab/dtd/train_ssf.sh and found that the result could not match the number on the paper.  At the end of the 100 epoch, the top 1 accuracy is 67.44 instead of 75.1 reported on the paper. I can reproduce the performance of most of the VTAB datasets. I'm curious if I missed anything for DTD or if there are mistakes in the training script for DTD. Thank you so much again and I look forward to hearing back from you.
